local.settings.example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "",
+    "FUNCTIONS_WORKER_RUNTIME": "node",
+    "DATABASE_URI": "MongoDB Connection String Here",
```

Comment:
Not in the scope of youre review, but it would be nice to implement Azure Key Vault here so we can commit safely to github without needing to hide our connection strings